1-js/05-data-types/02-number/article.md

```diff
@@ -50,7 +50,7 @@ let mсs = 0.000001;
 Just like before, using `"e"` can help. If we'd like to avoid writing the zeroes explicitly, we could write the same as:
 
 ```js
-let mcs = 1e-6; // five zeroes to the left from 1
+let mcs = 1e-6; // six zeroes to the left from 1
```

Comment:
not helpful. perhaps:

`let mcs = 1e-6; // five zeroes to the left of 1 (to the right of the decimal point)`

Reply:
There is no such “up to the decimal point” condition here. Besides, literally after this sentence comes the following: 

> If we count the zeroes in `0.000001`, there are **6** of them. So naturally it's `1e-6`.

Reply:
then fix that:

> > If we count the zeroes in `0.000001`, there are **5** of them after the decimal point. So naturally it's `1e-6`.